Add read/write of disabled controls to input file

LRossman opened the pull request:
Addresses issue #822.

eladsal commented:
Looks good. Thank you @LRossman .

Do we need to add something to `ReleaseNotes2_3.md`?

LRossman commented:
On my local branch I added the following sentence to the existing Release Notes item that first mentioned the additional API functions for enabling/disabling controls:
`A new keyword `DISABLED` is added to the end of a control or rule statement in an EPANET input file to indicate that the control is disabled.`
But I forgot to include the updated Notes file in this PR. I will do so at a later date when I will also re-format the notes into sections (New Features, Updated Features, Bug Fixes) to make it easier to read.
